build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -258,6 +258,11 @@ def convert_target(target):
   if m:
     return ["iree::samples::" + _convert_bazel_path(m.group(1))]
 
+  # Map //tools/(.*) -> \1
+  m = re.match("^//tools[/|:](.+)", target)
```

Comment:
Oh no :fearful: I just saw this collection of regexes. This certainly complicates this script quite a bit and points to increased complexity in our package hierarchy. So now everything under tools just has an unqualified name?

Reply:
Yeah. I think @stellaraccident had some more thoughts on how to organize the target namespaces after the move settles. In this case I intentionally wanted tools to use no package prefix, so they appear at the top level (still with `iree-` in their names), next to other tools like `mlir-opt` and `llvm-tblgen`